endpoints/net.go

```diff
@@ -0,0 +1,6 @@
+package endpoints
+
+type TlsConfig struct {
+	CertPath string
+	KeyPath  string
+}
```

Comment:
Might be a little better to call this file `config.go`.

Reply:
changed the file name